lib/dotcom_web/components/system_status/commuter_rail_status.ex

```diff
@@ -131,127 +130,79 @@ defmodule DotcomWeb.Components.SystemStatus.CommuterRailStatus do
-        <.icon name="chevron-right" class="h-3 w-2 fill-gray-dark ml-3 mr-2 shrink-0" />
-      </div>
-    </a>
+    <.row
+      label="No Scheduled Service"
+      row_name={row_name(@status)}
+      status={:no_scheduled_service}
+      url={@status.url}
+    />
     """
   end
 
   # The 'normal' case where there are no alerts.
   # We show a row indicating "Normal Service".
-  defp row(%{row: %{alert_counts: alert_counts}} = assigns) when alert_counts == %{} do
+  defp rows_for_line(%{status: %{alert_counts: alert_counts}} = assigns)
+       when alert_counts == %{} do
     ~H"""
-    <a
-      class={[
-        "flex items-stretch",
-        "hover:bg-brand-primary-lightest cursor-pointer group/row",
-        "text-black no-underline font-normal",
-        "min-h-12"
-      ]}
-      href={@row.url}
-      data-test="status-row"
-    >
-      <div class="grid items-stretch grid-cols-[min-content_min-content_auto] grow">
-        <div class="flex items-center pl-1 py-2 min-w-32 sm:min-w-48 md:min-w-56 lg:min-w-72 text-md md:text-lg border-t-xs border-gray-lightest">
-          <span>{row_name(@row)}</span>
-        </div>
-
-        <div class="px-2 flex items-center border-t-xs border-gray-lightest">
-          <.status_icon status={:normal} />
-        </div>
-
-        <div class="grow flex items-center py-2 border-t-xs border-gray-lightest">
-          <.status_label status={:normal} />
-        </div>
-      </div>
-      <div class="self-stretch flex items-center border-t-xs border-gray-lightest">
-        <.icon name="chevron-right" class="h-3 w-2 fill-gray-dark ml-3 mr-2 shrink-0" />
-      </div>
-    </a>
+    <.row label="Normal Service" row_name={row_name(@status)} status={:normal} url={@status.url} />
     """
   end
 
   # For cases where we have alerts, we have to show the first alert along with route information
   # and then show subsequent rows without the route information.
-  defp row(%{row: %{alert_counts: alert_counts}} = assigns) do
-    [first | rest] = combine_alert_counts(alert_counts)
-    assigns = assigns |> assign(:first, first) |> assign(:rest, rest)
-
+  defp rows_for_line(assigns) do
     ~H"""
-    <a
-      class={[
-        "flex items-stretch",
-        "hover:bg-brand-primary-lightest cursor-pointer group/row",
-        "text-black no-underline font-normal",
-        "min-h-12"
-      ]}
-      href={@row.url}
-      data-test="status-row"
-    >
-      <div class="grid items-stretch grid-cols-[min-content_min-content_auto] items-center grow">
-        <div class="flex items-center pl-1 py-2 min-w-32 sm:min-w-48 md:min-w-56 lg:min-w-72 text-md md:text-lg border-t-xs border-gray-lightest">
-          <span>{row_name(@row)}</span>
-        </div>
+    <.row
+      :for={
+        {{status, label}, index} <-
+          @status.alert_counts |> combine_alert_counts() |> Enum.with_index()
+      }
+      disrupted
+      label={label}
+      row_name={(index == 0 && row_name(@status)) || nil}
```

Comment:
nit: could also do `if(index == 0, do: row_name(@status))`

Reply:
Huh, yeah I was trying that and confusing myself about where the parens() should go.

[This seems to work!](https://github.com/mbta/dotcom/pull/2576/commits/70416f86032642c3988ab5c79193e47b93fc10ae)